feature/signupflow/finishprofile/src/main/java/com/epmedu/animeal/signup/finishprofile/presentation/FinishProfileScreen.kt

```diff
@@ -24,6 +25,10 @@ fun FinishProfileScreen() {
     val state by viewModel.stateFlow.collectAsState()
     val focusRequester = remember { FocusRequester() }
 
+    if (state.networkState == NetworkState.Lost) {
+        navigator.popBackStackOrNavigate(SignUpRoute.Onboarding.name)
```

Comment:
Why did you decide to go back from Finish Profile instead of preventing going to Finish Profile from the Splash screen?

Reply:
It's possible, but from SplashScreen these are options: Splash, SignUp and Tabs. I would need to add parameters from SplashScreen to redirect to SignUp and then to the correct screen (OnBoardingScreen). Each screen has their own logic to move through the SignUp flow so I would need to block/cancel those using NetworkState. At the end it was easier to just redirect to SignUp from the conflicting screen.

Reply:
But we already have logic to navigate to the Finish Profile screen from the Splash screen. It's not managed by SignUpFlow. Check SplashViewModel